conans/test/integration/cache/backup_sources_test.py

```diff
@@ -76,35 +76,18 @@ def source(self):
 
     def test_upload_sources_backup(self):
         client = TestClient(default_server_user=True)
+        file_server = TestFileServer()
```

Comment:
There are many others backup-sources tests to migrate.
I wanted to validate that the new ``FileServer`` works well before doing the whole migration.